metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/bolt/ParserBolt.java

```diff
@@ -116,6 +122,9 @@ public void execute(Tuple tuple) {
       boolean ackTuple = !writer.handleAck();
       int numWritten = 0;
       if(sensorParserConfig != null) {
+        if (configUpdatedFlag.getAndSet(false)) {
+          parser.configurationUpdated(getSensorParserConfig().getParserConfig());
```

Comment:
Sorry I wasn't completely clear.  Line 125 would change to this:
parser.configurationUpdated(sensorParserConfig.getParserConfig());

So config is fetched in line 118 and used everywhere after that until the next execute.